### ISSUE TITLE
As a tester, I want to know how to  concatenate the results of a multi-step process (eg. with period, newline, etc.).

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#293]

**Comment by mcking65** [comment]
@robfentress 

Is this issue about capturing screen reader output in tests where multiple commands, e.g., multiple presses of next item key, were needed to perform the test?
Do you want to know how to separate the output for each of the commands?

**Comment by robfentress** [comment]
Yes.  For comparison of the results of testers and especially for the purposes of automation of tests, the way this is handled may be relevant.  But maybe not.  Perhaps, this being only the first step, it is just assumed that any results from the manual tests will still need to be massaged prior to writing any automated test in order to make them work.  

On a related note (separate ticket?), we should make clear whether to include in what we record any instructions provided by the screen reader that are only presented after a delay.  Perhaps the instructions should further specify how long to wait for any such instructions.  Why?  People who are not as experienced in the use of screen readers may otherwise be confused as to what is relevant information to be included and what is incidental to the purpose of the test.  I dunno -- maybe we're just assuming that as background knowledge for any testers?
